Patterns/Base-PhoneNumbers.yaml

```diff
@@ -1,7 +1,19 @@
 ---
 NumberReplaceToken: '@builtin.phonenumber'
-IntegerRegexDefinition: !paramsRegex
-  def: (((?<!\d+\s*)-\s*)|((?<=\b)(?<!(\d+\.|\d+,))))\d{1,3}({thousandsmark}\d{3})+(?={placeholder})
-  params: [ placeholder, thousandsmark  ]
-PlaceHolderDefault: \\D|\\b
+NoLetterPrefixRegex: !simpleRegex
+  def: (?<![a-z|0-9])
+NoLetterSuffixRegex: !simpleRegex
+  def: (?![a-z|0-9]) 
+GeneralPhoneNumberRegex: !nestedRegex
+  def: ({NoLetterPrefixRegex}((\d[\s-/]?){8,}){NoLetterSuffixRegex})
+  references: [ NoLetterPrefixRegex, NoLetterSuffixRegex ]
+BrazilPhoneNumberRegex: !nestedRegex
+  def: ({NoLetterPrefixRegex}(\(?\s?\d{2,3}\s?\)?\s?)\s?\d{4,5}-?\d{3,4}{NoLetterSuffixRegex})
+  references: [ NoLetterPrefixRegex, NoLetterSuffixRegex ]
+UkPhoneNumberRegex: !nestedRegex
+  def: ({NoLetterPrefixRegex}((00)|\+)?\s?\d{2}\s?(\(0\))?\d{3,5}[/-]?\s?(\d{7,8}|\d{3}\s?\d{3}){NoLetterSuffixRegex})
+  references: [ NoLetterPrefixRegex, NoLetterSuffixRegex ]
+GermanyPhoneNumberRegex: !nestedRegex
```

Comment:
If you changed the YAML file, there should be a .cs and a .ts files created from it.
Can you add the .ts file to this PR?

Reply:
OK, I'll add it.

Reply:
I think we should create a new package to contain it. For now, it doesn't exist. And the typescript part in our pattern document seems to be out of date.